src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Media.")]
 
 [assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.8.0")]
+[assembly: AssemblyFileVersion("2.0.0")]
```

Comment:
I believe there are 4 periods in assembly versioning in `AssemblyInfo.cs`, please look at an example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Cdn/Management.Cdn/Properties/AssemblyInfo.cs#L11)

Reply:
Changed and pushed.